doc/en/user/source/services/generic_vendor_options.txt

```diff
@@ -0,0 +1,29 @@
+General Vendor Options
+----------------------
+
+These vendor options are available for all operations.
+
+content-disposition
+^^^^^^^^^^^^^^^^^^^
+
+The ``content-disposition`` parameter directs how a web browser directed to handle returned content. The syntax is::
+
+   content-dispoition=<dispoition>
+
+Where ``content-dispoition=attachment`` to direct the browser to save the content to disk (even if it is a format such as json or text which it may be wlling to display directly).
+
+Where ``content-dispoition=inline`` asks the browser to display the content. Note this may present a difficulty when asked to display large content.
```

Comment:
```suggestion
Where ``content-disposition=inline`` asks the browser to display the content. Note this may present a difficulty when asked to display large content.
```

"present a difficulty "  is rather vague... how about 
```suggestion
Where ``content-disposition=inline`` asks the browser to display the content. Note this may present performance issues when asked to display very large content.
```